CMakeLists.txt

```diff
@@ -84,6 +84,9 @@ if(SFML_BUILD_WINDOW)
     sfml_set_option(SFML_OPENGL_ES ${OPENGL_ES} BOOL "TRUE to use an OpenGL ES implementation, FALSE to use a desktop OpenGL implementation")
 endif()
 
+# add an option for building the test suite
+sfml_set_option(SFML_BUILD_TEST TRUE BOOL "TRUE to build the SFML test suite, FALSE to ignore them")
```

Comment:
> SFML_BUILD_TEST

`SFML_BUILD_TESTS` or `SFML_BUILD_TEST_SUITE` would be more correct.

> FALSE to ignore them

to ignore _it_ 😉